wicket-core/src/main/java/org/apache/wicket/resource/CssUrlReplacer.java

```diff
@@ -43,6 +49,8 @@
 	// The pattern to find URLs in CSS resources
 	private static final Pattern URL_PATTERN = Pattern.compile("url\\(['|\"]*(.*?)['|\"]*\\)");
 
+	private static final String EMBED_BASE64 = "embedBase64";

```

Comment:
Please make this `public`.
An application developer may want to use ...
And add javadoc what it is about.